base-notebook/Dockerfile.ppc64le

```diff
@@ -0,0 +1,105 @@
+# Copyright (c) IBM Corporation 2016 
```

Comment:
we leave the license as BSD so that it's consistent with the rest of the repository